docs/modules/user-guide/partials/proc_adding-plugin-component-to-a-devfile.adoc

```diff
@@ -14,74 +14,33 @@ This section describes how to add a `plugin` component to a devfile.
 
 . Define a component using the type `plugin`.
 
-. Define the `id` attribute. It is slash-separated publisher, name and version of plug-in from the Plug-in registry. List of available {prod-short} plug-ins and more information about registry can be found in the link:https://github.com/eclipse/che-plugin-registry[{prod-short} plug-in registry] GitHub repository.
+. Define the `id` attribute. It is the stack name in a devfile v2 registry. Devfile v2 registry should follow the convention according to the link:https://github.com/devfile/api/blob/master/docs/proposals/registry/index-server-REST-API.md[proposal].
```

Comment:
This is referred to as Devfile 2.0 In the README.md for [devfile/api](https://github.com/devfile/api). This implies to me that we should not refer to it as "Devfile v2" or "devfile v2".

Reply:
Yes, let's defer. I'm addressing language concerns in #39. In this PR, I'm working on a consistent way of referring to devfile 2.0. I'll address this concern when I resume a massive language review of the docs. If I'm not mistake, I think we want to refer to it as v2.0 devfile.